packages/wallet-lib/src/types/Storage/methods/rehydrateState.js

```diff
@@ -17,7 +17,11 @@ const rehydrateState = async function rehydrateState() {
           Object.keys(wallets).forEach((walletId) => {
             const walletStore = this.getWalletStore(walletId);
             if (walletStore) {
-              walletStore.importState(wallets[walletId]);
+              try {
```

Comment:
Please wrap the whole `forEach` into `try ... catc ...`, and remove both `wallets` and `chains` from the adapter. 

Same for `forEach` with chains below